app/models/spree/order_decorator.rb

```diff
@@ -191,5 +191,48 @@ def flow_payment_method
         'cc' # creait card is default
       end
     end
+
+    def flow_customer_email
```

Comment:
Are the rest of flow methods in `order_decorator`? 😕

Reply:
Yep.

Reply:
You think it would be better to move those methods outside the decorator? Maybe would make more sense to have another entity like FlowOrder which should store those methods? 

The only issue with that is that it would require a little refactor. Probably we can handle that later, but not sure if it would be worth doing now due to short times :S